assets/lib/VERSION

```diff
@@ -24,3 +24,5 @@ meting@2.0.1 https://github.com/metowolf/MetingJS
 gitalk@1.6.2 https://github.com/gitalk/gitalk
 valine@1.4.14 https://valine.js.org/
 cookieconsent@3.1.1 https://github.com/osano/cookieconsent
+twikoo@1.4.1 https://github.com/imaegoo/twikoo
```

Comment:
@OnlyFor did you used some packaging tool or updated this file manually? I'm asking this because I don't know how this file was generated ))

Reply:
I……manually update this file……

Reply:
No worries. I just asked. I thought maybe there is some package manager tool.